CMakeLists.txt

```
@@ -18,5 +18,39 @@ add_executable("${PROJECT_NAME}" main.cpp)
 target_link_libraries("${PROJECT_NAME}" cpackexamplelib)
 target_link_libraries(cpackexamplelib Boost::filesystem ${YAML_CPP_LIBRARIES})
 
+set_target_properties(cpackexamplelib
+  PROPERTIES
+    PUBLIC_HEADER filesystem/filesystem.hpp
+    PUBLIC_HEADER flatset/flatset.hpp
+    PUBLIC_HEADER yamlParser/yamlParser.hpp
+  )
```

```suggestion
set_target_properties(cpackexamplelib PROPERTIES PUBLIC_HEADER "fem/fem.hpp;filesystem/filesystem.hpp;flatset/flatset.hpp;yamlParser/yamlParser.hpp")
```
Otherwise, `PUBLIC_HEADER` gets overwritten and only `yamlParser.hpp` gets installed.